src/main/steps/c100-rebuild/check-your-answers/CheckYourAnswersGetController.ts

```diff
@@ -0,0 +1,29 @@
+import autobind from 'autobind-decorator';
+import { Response } from 'express';
+
+import { FieldPrefix } from '../../../app/case/case';
+import { C100_CASE_EVENT } from '../../../app/case/definition';
+import { AppRequest } from '../../../app/controller/AppRequest';
+import { GetController, TranslationFn } from '../../../app/controller/GetController';
+
+@autobind
+export default class CheckYourAnswersGetController extends GetController {
+  constructor(
+    protected readonly view: string,
+    protected readonly content: TranslationFn,
+    protected readonly fieldPrefix: FieldPrefix
+  ) {
+    super(view, content);
+  }
+
+  public async get(req: AppRequest, res: Response): Promise<void> {
+    await req.locals.C100Api.updateCase(
+      // eslint-disable-next-line @typescript-eslint/no-non-null-assertion
+      req.session.userCase!.caseId!,
+      req.session.userCase,
+      req.originalUrl,
```

Comment:
@yogendraU We need to save check your answers URL here I think so that user will be redirected to check your answers when comes back.

Reply:
It's check your answers URL only as it's get controller.